core/include/usModuleRegistry.h

```diff
@@ -27,9 +27,11 @@
 #include <string>
```

Comment:
On second thought, making usModuleRegistry.h private should be done at the same time and doesn't need to wait until ModuleInfo is refactored.